docs/blocks/avoiding-properties-limit.md

```diff
@@ -50,6 +50,85 @@ And your conditions like this;
 
 "condition": "q.block_property('wiki:division') == 1 && q.block_property('wiki:value') == 2"
 ```
+
+# Binary Method
+We can use two properties to produce a single number that can range from 0 to 255.
+
+In same way we can use 3 properties to produce a single number that can range from 0 to 4095
```

Comment:
Please change it to `Using the same method, we can use 3 properties to produce a single number that can range from 0 to 4095.`